### ISSUE TITLE
[Rust] Syntax warnings aren't printed out when using the Rust CLI

### ISSUE THREAD

**Issue by seanchen1991** [osohq/oso#802]
For example, with the following Polar file:

```
# foo.polar
allow(actor: String, "GET", expense: Expense) if
    actor.ends_with("@example.com");
```

Passing it as a CLI arg to `python3 -m oso foo.polar` outputs the following warnings:

```
[warning] Singleton variable expense is unused or undefined, see <https://docs.osohq.com/using/polar-syntax.html#variables>
002: allow(actor: String, "GET", expense: Expense) if
                                 ^
[warning] Unknown specializer Expense
002: allow(actor: String, "GET", expense: Expense) if
                                          ^
```

However, passing it via `oso foo.polar` outputs no warnings.

**Comment by saolsen** [comment]
Hey, thanks for this.

I believe it has to do with how we've configured logging for the repl process. In the rust library those lines get logged with the logging crate as warn statements but I think we've mis-configured the repl process to filter them out.

Until we fix it you can see the warnings by setting the log level via an env var. `RUST_LOG=warn`

`RUST_LOG=warn oso foo.polar`

**Comment by gj** [comment]
@gkaemmer was this fixed by #1115?

**Comment by gkaemmer** [comment]
Yeah, indeed it was -- warnings will now be logged with `std::eprintln!` instead of `tracing::warn`, so they'll show up directly on stderr.
